docs/content/learn/tutorials/general/extension.md

```diff
@@ -483,8 +484,8 @@ Links in OSCAL provide a means to reference an arbitrary resource, which allows
 
 A link can:
 
-1. Reference (external) information that is not represented in OSCAL format.  This could include references to (cybersecurity) laws and regulations, references to organizational standards and guides, references to a software bill of materials (SBOM), and more.
-2. Reference objects within the current OSCAL document.
+1. Reference (external) information that is not represented in OSCAL format.  This could include references to (cybersecurity) laws and regulations, references to organizational standards and guides, references to a software bill of materials (SBOM), and more. See [*linking to a network resolvable resource](/concepts/uri-use/#linking-to-a-network-resolvable-resource) for more information.
```

Comment:
```suggestion
1. Reference (external) information that is not represented in OSCAL format.  This could include references to (cybersecurity) laws and regulations, references to organizational standards and guides, references to a software bill of materials (SBOM), and more. See [*linking to a network resolvable resource*](/concepts/uri-use/#linking-to-a-network-resolvable-resource) for more information.
```